### PR TITLE
Minor tweaks to Android client

### PR DESCRIPTION
**Pull request by kerneis-anssi**

### CODE REVIEW

File: clients/Android/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+This should display a QR Code.
+
+Start the client app on your phone (from Android Studio), and follow the
+instructions displayed by the server to scan the QR Code.
+
+## Troubleshooting
+
+General tips Android Studio tips, not specific to Ultrablue:
```
Comment (ghost) [inline clients/Android/README.md]: Did you meant "General Android studio tips"?

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by kerneis-anssi** [comment]
Please take another look.

**Comment by kerneis-anssi** [comment]
@lfb-anssi can you please confirm this is okay to merge for you?

**Review by ghost — APPROVED** [review]
Looks good to me!

**Pull request merged**